### PR TITLE
Feature/home page module

### PR DESCRIPTION
**Pull request by Secondtwin**

### CODE REVIEW

File: src/app/home-page/home-page-cards/home-page-cards.component.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
+@import "../../variables.scss";
+
+@mixin cards($col) {
```
Comment (SteveRusin) [inline src/app/home-page/home-page-cards/home-page-cards.component.scss]: declare mixins in separate file

File: src/app/home-page/home-page.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="main-background">
+    <img class="img-pikachu" src="../../assets/pokemons/pikachu.png"/>
+    <!-- <img class="img-pokeball" src="../../assets/pokemons/pokeball.png"/> -->
+    <div class="main-page-desc">
+        <h2>Pokemania</h2>
+        <p>This game is developed for pokemon lovers. Here you can choose your favorite pokemon
+          and fight as a team against other pokemons</p>
+    </div>
+    <div class="wrapper-btn">
+      <svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="19em" height="18em" viewBox="0 0 3490 3000" preserveAspectRatio="xMidYMid meet">
```
Comment (SteveRusin) [inline src/app/home-page/home-page.component.html]: move this svg to separate file

File: src/app/home-page/home-page.module.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { HomePageComponent } from './home-page.component';
+import { HomePageCardsComponent } from './home-page-cards/home-page-cards.component';
+
+import { homeRoutes } from './home-page-routing.module';
+
+import { MatAutocompleteModule,
+  MatButtonModule,
```
Comment (SteveRusin) [inline src/app/home-page/home-page.module.ts]: do you really need all this stuff?

File: src/app/home-page/home-page-cards/home-page-cards.component.scss (hunk 1 above)
Comment (DariiaKuzovkova) [reply src/app/home-page/home-page-cards/home-page-cards.component.scss]: done

File: src/app/home-page/home-page.module.ts (hunk 3 above)
Comment (DariiaKuzovkova) [reply src/app/home-page/home-page.module.ts]: nope) checked and removed some of them

**Pull request merged**